addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/ReplacerParam.java

```diff
@@ -108,6 +113,22 @@ public class ReplacerParam extends VersionedAbstractParam {
                             true,
                             REPORT_TO_REPLACEMENT,
                             List.of(),
+                            false),
+                    new ReplacerParamRule(
+                            MODIFIED_SINCE_DESC,
+                            ReplacerParamRule.MatchType.REQ_HEADER,
+                            HttpFieldsNames.IF_MODIFIED_SINCE,
+                            false,
+                            "",
+                            List.of(),
+                            false),
+                    new ReplacerParamRule(
+                            NONE_MATCH_DESC,
+                            ReplacerParamRule.MatchType.REQ_HEADER,
+                            HttpFieldsNames.IF_NONE_MATCH,
+                            false,
+                            "",
+                            List.of(),
```

Comment:
Would it be more correct/efficient to pass null here instead of instantiating an empty list? (It seems to behave the same way ultimately.)

Reply:
it will not make much difference.